charts/argocd-workflows-apps/templates/monitoring-stack-application.yaml

```diff
@@ -0,0 +1,32 @@
+{{- if .Values.monitoring.enabled }}
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: monitoring
+  namespace: workflows
+  annotations:
+    argocd.argoproj.io/sync-wave: "2"
+spec:
+  destination:
+    namespace: workflows
+    server: {{ .Values.destination.server }}
+  project: workflows
```

Comment:
I believe this can (and should) be templated using the namespace, since we map projects to namespaces in a one-to-one fashion